csharp/test/Apache.Arrow.Tests/ArrowFileReaderTests.cs

```diff
@@ -47,6 +48,55 @@ public void Ctor_LeaveOpenTrue_StreamValidOnDispose()
             Assert.Equal(0, stream.Position);
         }
 
+        [Fact]
+        public async Task Ctor_MemoryPool_AllocatesFromPool()
+        {
+            await Ctor_MemoryPool_AllocatesFromPoolHelper(
```

Comment:
nit: you can just return the task without awaiting it (the unit test runner should do it)

Reply:
With the above theory, I eliminated this helper method all together.